{"created": 1759385913.587662, "duration": 3.1004481315612793, "exitcode": 2, "root": "C:\\Users\\Sk Samdan\\Desktop\\github\\digitalmodel", "environment": {}, "summary": {"total": 0, "collected": 0}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "tests/modules/marine_analysis/python_mathcad/test_mathcad_validation.py", "type": "Module"}]}, {"nodeid": "tests/modules/marine_analysis/python_mathcad/test_mathcad_validation.py", "outcome": "failed", "result": [], "longrepr": "ImportError while importing test module 'C:\\Users\\Sk Samdan\\Desktop\\github\\digitalmodel\\tests\\modules\\marine_analysis\\python_mathcad\\test_mathcad_validation.py'.\nHint: make sure your test modules/packages have valid Python names.\nTraceback:\n..\\..\\..\\AppData\\Roaming\\uv\\python\\cpython-3.11.13-windows-x86_64-none\\Lib\\importlib\\__init__.py:126: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nC:\\Users\\Sk Samdan\\Desktop\\github\\digitalmodel\\tests\\modules\\marine_analysis\\mathcad-to-python-passing_ship\\test_mathcad_validation.py:37: in <module>\n    ???\nE   ImportError: cannot import name 'mathcad_to_python_passing_ship' from 'digitalmodel.modules.marine_analysis' (C:\\Users\\Sk Samdan\\Desktop\\github\\digitalmodel\\src\\digitalmodel\\modules\\marine_analysis\\__init__.py)"}], "tests": []}